doc/rbd/rbd-mirroring.rst

```diff
@@ -0,0 +1,250 @@
+``mirror image disable`` command along with the pool and image name::
+
+        rbd mirror image disable {pool-name}/{image-name}
+
+For example::
+
+        rbd --cluster local mirror image disable image-pool/image-1
+
+Image Promotion and Demotion
+----------------------------
+
+In a failover scenario where the primary designation needs to be moved to the
+image in the peer Ceph cluster, access to the primary image should be stopped
+(e.g. power down the VM or remove the associated drive from a VM), demote the
+current primary image, promote the new primary image, and resume access to the
+image on the alternate cluster.
+
+.. note:: RBD only provides the necessary tools to facilitate an orderly
+   failover of an image. An external mechanism is required to coordinate the
+   full failover process (e.g. closing the image before demotion).
+
+To demote an image to non-primary with ``rbd``, specify the
+``mirror image demote`` command along with the pool and image name::
+
+        rbd mirror image demote {pool-name}/{image-name}
+
+For example::
+
+        rbd --cluster local mirror image demote image-pool/image-1
+
+To promote an image to primary with ``rbd``, specify the ``mirror image promote``
+command along with the pool and image name::
+
+        rbd mirror image promote {pool-name}/{image-name}
+
+For example::
+
+        rbd --cluster remote mirror image promote image-pool/image-1
+
+.. tip:: Since the primary / non-primary status is per-image, it is possible to
+   have two clusters split the IO load and stage failover / failback.
+
+.. note:: Promotion can be forced using the ``--force`` option.  Forced
+   promotion is needed when the demotion cannot be propagated to the peer
+   Ceph cluster (e.g. Ceph cluster failure, communication outage).  This will
+   result in a split-brain scenario between the two peers and the image will no
+   longer be in-sync until a `force resync command`_ is issued.
+
+Force Image Resync
+------------------
+
+If a split-brain event is detected by the ``rbd-daemon``, it will not attempt
+to mirror the affected image until corrected.  To resume mirroring for an image,
+first `demote the image`_ determined to be out-of-date and then request a resync
+to the primary image.  To request an image resync with ``rbd``, specify the
+``mirror image resync`` command along with the pool and image name::
+
+        rbd mirror image resync {pool-name}/{image-name}
+
+For example::
+
+        rbd mirror image resync image-pool/image-1
+
+.. note:: The ``rbd`` command only flags the image as requiring a resync. The
+   local cluster's ``rbd-mirror`` daemon process is responsible for performing
+   the resync asynchronously.
+
+rbd-mirror Daemon
+=================
+
+The ``rbd-mirror`` daemon is responsible for watching image journals on a
+remote, peer cluster and replaying the journal events against the local
+cluster.  The RBD image journaling feature records all modifications to the
```

Comment:
I know there are two, but the phrase "The rbd-mirror daemon" causes the reader to think in terms of a single daemon. How about this:

The two `rbd-mirror` daemons (one on each peer cluster) are responsible for watching image journals on the remote, peer cluster and . . .